client/components/domains/featured-domain-suggestions/index.jsx

```diff
@@ -112,6 +112,7 @@ export class FeaturedDomainSuggestions extends Component {
 						isSignupStep={ this.props.isSignupStep }
 						uiPosition={ 0 }
 						fetchAlgo={ this.getFetchAlgorithm( primarySuggestion ) }
+						buttonStyles={ { primary: true } }
```

Comment:
Don't think we need this. If we can derive these styles just from the `isFeatured` flag, I'd rather we do that in the component.